### PR TITLE
Touch cmd (without flags for now)

### PR DESCRIPTION
**Pull request by rossem**
- added shlex library to intelligently parse tokens
- improved tests

### CODE REVIEW

**Comment by rossem** [comment]
Yep, definitely. The tests are just so we don't flood main with random shit and make sure stuff works

File: parse/parse.go [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
 
 	return models.Command{
 		RawCommandString: strings.Join(tokens, " "), // remove unneeded spaces
 		BaseCommand:      tokens[0],
 		Tokens:           tokens,
+	}, nil
+}
+
+func IsValidFilePath(path string) bool {
+	if _, err := os.Stat(path); err != nil {
```
Comment (Wojtechnology) [inline parse/parse.go:25]: fuck off with this fancy one liner shit

File: test/nope_test.go [hunk 2]
```diff
@@ -9,23 +10,37 @@ import (
...
+	}
 
-func TestCd(t *testing.T) {
-	undo, err := testCmd("cd ..")
+	undo, err := match.GetUndoCommand(cmd)
 	if err != nil {
-		t.Errorf("Cd command failed")
+		t.Errorf(formatError(rawCmd, "could not match command"))
 	}
 	log.Println(undo)
```
Comment (Wojtechnology) [inline test/nope_test.go:23]: Test for equality instead of just printing.

**Comment by Wojtechnology** [comment]
lgtm

**Pull request merged**